modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-kubernetes-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="k8s-nmstate-uninstall-operator_{context}"]
+= Uninstalling the Kubernetes NMState Operator
+
+You can use the {olm-first} to uninstall the Kubernetes NMState Operator, but by design {olm} does not delete any associated custom resource definitions (CRDs), custom resources (CRs), or API Services. 
+
+Before you uninstall the Kubernetes NMState Operator from the `Subcription` resource that {olm} uses, identify what Kubernetes NMState Operator resources to delete. This identification ensures that you can delete the resources without impacting your running cluster.
+
+.Procedure
+
+. Find `openshift-nmstate` namespace that exists in the `Subcription` resource that {olm} uses by running the following command:
++
+[source,terminal]
+----
+$ oc get --all-namespaces subscription
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE           NAME                      	  PACKAGE                   	SOURCE                   	 CHANNEL
+openshift-nmstate   kubernetes-nmstate-operator   kubernetes-nmstate-operator   kubernetes-nmstate-catalog   stable
+...
+----
+
+. Unsubscribe the Kubernetes NMState Operator from the `Subcription` resource by running the following command:
++
+[source,terminal]
+----
+$ oc delete --namespace openshift-nmstate subscription kubernetes-nmstate-operator
+----
+
+. Find the `ClusterServiceVersion` (CSV) resource that associates with the Kubernetes NMState Operator by running the following command:
```

Comment:
Same with this step. Would the CSV exist anyway? So is something unique likely for finding it?

Reply:
This one is slightly trickier, because as you can see, the CSV name -- `kubernetes-nmstate-operator.v4.18.0` -- contains the version.